changelog.json

```
@@ -1,5 +1,8 @@
 {
   "releases": {
+    "2.6.3-beta2": [
+      "[Improved] Update app icon - #10855 #7536"
```

I'd mention this is a Big Sur specific icon.
```suggestion
      "[Improved] Update app icon for macOS Big Sur - #10855
```